lmdeploy/pytorch/engine/engine.py

```diff
@@ -353,6 +363,9 @@ def __init__(self,
         self._start_loop()
         self._loop_main = None
 
+        # for migration loop management
+        self.migration_event = asyncio.Event()
```

Comment:
The engine is lazy started since we might not have the event loop when creating engine.
I don't know if it is safe to initialize `asyncio.Event` here.